README.md

```diff
@@ -1 +1,61 @@
-# snet-daemon
\ No newline at end of file
+# snet-daemon
+
+SingularityNET Daemon
+
+## Getting Started
+
+These instructions are intended to facilitate the development and testing of SingularityNET Daemon. Users interested in
+deploying SingularityNET services using SingularityNET Daemon should install the appropriate binary as
+[released](#release).
+
+### Prerequisites
+
+* [Go 1.10+](https://golang.org/dl/)
+* [Dep 0.4.1+](https://github.com/golang/dep#installation)
+* [Node 8+ w/npm](https://nodejs.org/en/download/)
```

Comment:
Should note here that Node 10 is *not* supported due to package issues with sha3